manifests/k/KDE/KDEConnect/21.04.2.640/KDE.KDEConnect.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: KDE.KDEConnect
+PackageVersion: 21.04.2.640
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://binary-factory.kde.org/job/kdeconnect-kde_Release_win64/640/artifact/kdeconnect-kde-21.04.2-640-windows-msvc2019_64-cl.exe
+  InstallerSha256: B89AF8750D40C622FF542FF18F3DBDFE9BD0C3202FF423D24B68E5E4A3F53904
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
```

```suggestion
- Architecture: x64
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://binary-factory.kde.org/job/kdeconnect-kde_Release_win64/640/artifact/kdeconnect-kde-21.04.2-640-windows-msvc2019_64-cl.exe
  InstallerSha256: B89AF8750D40C622FF542FF18F3DBDFE9BD0C3202FF423D24B68E5E4A3F53904
  InstallerSwitches:
    Custom: /ALLUSERS
- Architecture: x64
  InstallerType: nullsoft
  Scope: user
  InstallerUrl: https://binary-factory.kde.org/job/kdeconnect-kde_Release_win64/640/artifact/kdeconnect-kde-21.04.2-640-windows-msvc2019_64-cl.exe
  InstallerSha256: B89AF8750D40C622FF542FF18F3DBDFE9BD0C3202FF423D24B68E5E4A3F53904
  InstallerSwitches:
    Custom: /CURRENTUSER
```
Knew this one looked suspicious 👀 Default installation is also for all users